base/functional/callback_helpers.cc

```diff
@@ -4,22 +4,29 @@
 
 #include "base/functional/callback_helpers.h"
 
+#ifndef _WIN32
 #include <dlfcn.h>
-
-static void* gRecordReplayValueFn;
+#else
+#include <windows.h>
+#endif
+
+static void* LookupRecordReplaySymbol(const char* name) {
+#ifndef _WIN32
+  void* fnptr = dlsym(RTLD_DEFAULT, name);
+#else
+  HMODULE module = GetModuleHandleA("windows-recordreplay.dll");
```

Comment:
Can we use this as a push toward better code sharing between all our runtimes and runtime repos?
We are copy-and-pasting so much of this 😢 
And then, sometimes, there are some unnecessary changes between them, e.g. some names are different in `skia`, not sure if that is all that necessary?

Reply:
It can be pretty difficult when working on this stuff to figure out exactly how to interface with chromium's build system so that we can avoid all code duplication.  I'd rather err on the side of expediency, these are simple stubs and copy-pasting them is an appropriate option to get things to work.  As part of this PR on windows we dynamically look up symbols defined in the same binary in record_replay.cc instead of defining them as externs, because there are many binaries on windows that link against base but not V8.  We could probably do the same thing for linux/mac and eliminate many of these dynamic lookups in other files, but that sort of cleanup isn't appropriate for this issue.

Reply:
Makes sense. I added a comment to the "Code Sharing" item in our Runtime 2023 planning doc, for Q3+.